core/services/ocr2/delegate.go

```diff
@@ -437,7 +437,7 @@ func (d *Delegate) newServicesMercury(
 	lc ocrtypes.LocalConfig,
 	ocrLogger commontypes.Logger,
 ) ([]job.ServiceCtx, error) {
-	if jb.OCR2OracleSpec.FeedID == (common.Hash{}) {
+	if jb.OCR2OracleSpec.FeedID == nil {
```

Comment:
Do you want to keep "equals nil OR equals zero" here?

Reply:
Zero value should not pop up by accident now its nullable